core/common/src/rpc/core/RpcRequest.ts

```diff
@@ -41,7 +41,7 @@ export class ResponseLike implements Response {
   public get status() { return 200; }
   public get statusText() { return ""; }
   public get trailer(): Promise<Headers> { throw new IModelError(BentleyStatus.ERROR, "Not implemented."); }
-  public get type(): ResponseType { return "basic"; }
+  public get type(): "basic" | "cors" | "default" | "error" | "opaque" | "opaqueredirect" { return "basic"; }
```

Comment:
do we need this union type when we are returning a literal?

Reply:
technically narrowing the return type would be a breaking change (someone might have extended the class to return something else)